src/pages/billing/Project.js

```diff
@@ -564,29 +564,39 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
 
   const tabToTable = {
     workspaces: h(Fragment, [
-      div({ role: 'table', 'aria-label': `workspaces in billing project ${billingProject.projectName}` }, [
-        h(WorkspaceCardHeaders, {
-          needsStatusColumn: billingAccountsOutOfDate,
-          sort: workspaceSort,
-          onSort: setWorkspaceSort
-        }),
-        div({}, [
-          _.flow(
-            _.orderBy([workspaceSort.field], [workspaceSort.direction]),
-            _.map(workspace => {
-              const isExpanded = expandedWorkspaceName === workspace.name
-              return h(WorkspaceCard, {
-                workspace: { ...workspace, billingAccountDisplayName: billingAccounts[workspace.billingAccount]?.displayName },
-                billingProject,
-                billingAccountStatus: billingAccountsOutOfDate && getBillingAccountStatus(workspace),
-                key: workspace.workspaceId,
-                isExpanded,
-                onExpand: () => setExpandedWorkspaceName(isExpanded ? undefined : workspace.name)
+      isOwner && _.isEmpty(workspacesInProject) ?
+        div({ style: { margin: '1rem' } }, [
+          h2({ style: { fontSize: 18, fontWeight: 600, lineHeight: '22px' } }, ['Billing project successfully created']),
+          h(ButtonPrimary, {
+            style: { padding: '1rem 1rem', textTransform: 'none' },
+            onClick: () => {
+              Nav.goToPath('workspaces')
+            }
+          }, ['Go to the workspace list page'])
```

Comment:
Sorry, I wasn't really aware of this yet to discuss it at our meeting this morning.

@admull why are we showing this? It's not like they will have any workspaces…. Is the point to get them to create one? If so, can we change the wording to make it clearer? I think a link would be better than a button, it should be something like "Go to the workspaces page to create your first workspace".

Also note this message is going to be shown *every* time the user goes to the billing project page until they have created a workspace. Therefore "Billing project successfully created" isn't a great thing to show unconditionally, given that the message will be persistent.

![image](https://user-images.githubusercontent.com/484484/198083763-52d1c62d-288c-47a4-83c0-733cbd133709.png)

Reply:
I think it would be great if we can come up with something that we are comfortable showing _whenever_ the workspaces list is empty. Because really it's useful information to anyone (for example, if you come to the page and have a shared billing project that you haven't created any workspaces yet in).